### ISSUE TITLE
ORCA crash when quitting experiment - suspected ELLIE issue

### ISSUE THREAD

**Issue by fbdescamps** [snoplus/orca#154]
To reproduce:
 - have ORCA sno+ running
 - save and close experiment
 - enjoy the beachball

Reported error:
```
2016-08-27 14:23:28.815 Orca[21101:548871] Failed to connect (vtMatrix) outlet from (ORFecDaughterCardController) to (NSMatrix): missing setter or instance variable
2016-08-27 14:29:31.231 Orca[21101:548871] *** Assertion failure in -[NSTextFieldCell _objectValue:forString:errorDescription:], /SourceCache/AppKit/AppKit-1343.16/AppKit.subproj/NSCell.m:1622
2016-08-27 14:29:31.241 Orca[21101:548871] *** Assertion failure in -[NSTextFieldCell _objectValue:forString:errorDescription:], /SourceCache/AppKit/AppKit-1343.16/AppKit.subproj/NSCell.m:1622
2016-08-27 14:29:31.258 Orca[21101:548871] *** Assertion failure in -[NSTextFieldCell _objectValue:forString:errorDescription:], /SourceCache/AppKit/AppKit-1343.16/AppKit.subproj/NSCell.m:1622
```

Screenshot

<img width="1870" alt="screen shot 2016-08-27 at 2 41 19 pm" src="https://cloud.githubusercontent.com/assets/1840217/18029400/32400bfa-6c65-11e6-9550-bbf82de1df07.png">

**Comment by EdLeming** [comment]
@fbdescamps. Thanks for the spot. This is a easy fix - see PR #156

**Comment by tlatorre-uchicago** [comment]
Fixed in #156.
